srtcore/sync.h

```diff
@@ -311,9 +311,9 @@ inline bool is_zero(const TimePoint<steady_clock>& t)
 ///////////////////////////////////////////////////////////////////////////////
 
 #if ENABLE_STDCXX_SYNC
-using Mutex = mutex;
-using UniqueLock = unique_lock<mutex>;
-using ScopedLock = lock_guard<mutex>;
+using Mutex = std::mutex;
+using UniqueLock = unique_lock<std::mutex>;
+using ScopedLock = lock_guard<std::mutex>;
```

Comment:
If `std::` prefix is needed for the mutex here, **maybe** `unique_lock` and `lock_guard` should be better using `std::` as well. 
And `using namespace std` in the head of `sync.h` should just be removed.